coalib/results/SourceRange.py

```diff
@@ -126,6 +126,51 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Creates a string representation of the SourceRange object
+        Here are doctest examples of the three different representations
+
+        >>> from os.path import abspath
+
+        >>> case1 = abspath('test_file')
+        >>> example_str1 = str(SourceRange.from_values('test_file', None, None,
+        ... None, None))
+        >>> case1 == example_str1
+        True
+
+        If the whole file is affected, then just the filename is shown.
+
+        >>> case2 = abspath('test_file') + ': L1 : L2'
+        >>> example_str2 = str(SourceRange.from_values('test_file', 1,
+        ... None, 2, None))
+        >>> case2 == example_str2
+        True
```

Comment:
I even tried this work around `# doctest: +ELLIPSIS` and `[...]`
Source - http://stackoverflow.com/questions/5812833/can-i-have-an-ellipsis-at-the-beginning-of-the-line-in-a-python-doctest